### PR TITLE
fetch show data 

### PR DESCRIPTION
**Pull request by mntedev**
I have completed the following tasks. 
1. added get data function to retrieve shows data from tv maze API.
2. added display shows data feature to display shows data on the home page on card layout 
3. added stsyling for cards

### CODE REVIEW

File: src/module/dom-utils.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+         <div class="card" id=${element.id}> 
+           
+           <img src=${element.image.medium} alt ="best" class="image"/>
+           <div class="show-info">
+            <p class="show-name"> ${element.name}</p>
+            <div class="like">
+            <i class="fa fa-heart heart" aria-hidden="true"></i>
+            <div class="likes-num">
+               <span> ${data.length}</span> likes
+            </div>
```
Comment (KaskMIL) [inline src/module/dom-utils.js:16]: Maybe after the likes-num div you can add a comments button to display the popup section, the rest looks nice

**Review by KaskMIL — CHANGES_REQUESTED** [review]
Looks so good so far, but maybe we can implement the change I suggest in the comment

**Comment by mntedev** [comment]
I have implemented your change request.

 1. I have added the comment button.
 
image

**Comment by KaskMIL** [comment]
That looks fantastic!

**Review by KaskMIL — APPROVED** [review]

**Pull request merged**